<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FreeFTPClass</class>
 <widget class="QWidget" name="FreeFTPClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FreeFTP</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditHost"/>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btnConnect">
       <property name="text">
        <string>连接</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditPassword"/>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="btnDisconnect">
       <property name="text">
        <string>断开</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelUser">
       <property name="text">
        <string>User</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelPassword">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelHost">
       <property name="text">
        <string>Host</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditUser"/>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="btnConnecting">
       <property name="text">
        <string>连接中...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="treeView"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="FreeFTP.qrc"/>
 </resources>
 <connections/>
</ui>
